plenum/server/consensus/ordering_service.py

```diff
@@ -2379,12 +2386,6 @@ def process_old_view_preprepare_request(self, msg: OldViewPrePrepareRequest, sen
         self._send(rep, dst=[replica_name_to_node_name(sender)])
 
     def process_old_view_preprepare_reply(self, msg: OldViewPrePrepareReply, sender):
-        # TODO: return the check after INDY-2238 about persisting of 3pc messages
```

Comment:
Why did we remove this comment? INDY-2238 is still not done....

Reply:
I checked the test in comment and discussed it with Renata and we decided to remove this comment.